source/issues-and-help.txt

```diff
@@ -0,0 +1,48 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant community of MongoDB .NET/C# Driver
```

Comment:
Per [this prior comment](https://github.com/mongodb/docs-csharp/pull/18#discussion_r980103407), use the source constant. Let me know if this doesn't make sense and we can discuss.

```suggestion
We are lucky to have a vibrant community of {+driver-long+}
```